kmk/extensions/keymap_extras/keymap_jp.py

```diff
@@ -0,0 +1,157 @@
+# What's this?　
+# This is a keycode conversion script. With this, KMK will work as a JIS keyboard.
+
+# Usage
+# ```python
+# from kmk.extensions.keymap_extras.keymap_jp import JP
+# keyboard.keymap = [ ... JP.AT ... ]
+# ```
+
+# Credit
+#  Proted from keymap_jp.h on QMK
+#  https://github.com/qmk/qmk_firmware/blob/master/quantum/keymap_extras/keymap_jp.h
+
+'''
+/* Copyright 2016 h-youhei
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http:#www.gnu.org/licenses/>.
+ *
+ * JP106-layout (Japanese Standard)
+ *
+ * For more information, see
+ * http://www2d.biglobe.ne.jp/~msyk/keyboard/layout/usbkeycode.html
+ * note: This website is written in Japanese.
+ */
+'''
+
+from kmk.keys import KC
+
+class JP:
+    '''
+    /*
+     * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐
+     * │Z↔︎H│ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │ - │ ^ │ ¥ │   │
+     * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┤
+     * │     │ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │ @ │ [ │     │
+     * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+     * │ Eisū │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; │ : │ ] │    │
+     * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┤
+     * │        │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │ \\ │      │
+     * ├─────┬──┴┬──┴──┬┴───┴┬──┴───┴──┬┴───┴┬──┴┬──┴┬──┴┬──┴┬─────┤
+     * │     │   │     │Muhen│         │ Hen │K↔H│   │   │   │     │
+     * └─────┴───┴─────┴─────┴─────────┴─────┴───┴───┴───┴───┴─────┘
+     */
+    '''
+
+    # Row 1
+    ZKHK = KC.GRV  # Zenkaku ↔︎ Hankaku ↔ Kanji (半角 ↔ 全角 ↔ 漢字)
```

Comment:
Is it possible that we add this into core KMK as another name for the GRV key instead of remapping it?